ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Config.hs

```diff
@@ -17,6 +17,7 @@ import           Ouroboros.Network.Magic (NetworkMagic)
 
 import           Ouroboros.Consensus.Block
 import           Ouroboros.Consensus.BlockchainTime
+import           Ouroboros.Consensus.Node.Config
```

Comment:
It feels dirty to import from `.Node` here, but it makes sense to have `BlockConfig` and `ConfigSupportsNode` in the same file, just look at the methods.

Reply:
See last comment about `SupportsNode`.